ansible/monitoring.yml

```diff
@@ -16,10 +51,10 @@
       logs_enabled: true
       hostname: "{{ ansible_fqdn }}"
       tags:
-        - "platform:{{ datadog_platform|default('unknown') }}"
-        - "region:{{ datadog_region|default('unknown') }}"
-        - "env:{{ datadog_env|default('unknown') }}"
-        - "color:{{ datadog_color|default('unknown') }}"
+        - "platform:{{ platform|default('os') }}"
+        - "region:{{ region|default('eu-east') }}"
```

Comment:
This default may be misleading.

Reply:
I know, but it’s really hard to work with openstack here, thus I hardcoded it until we ditch it soon